client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -37,16 +37,19 @@ function getUrlInfo( url: string ) {
 	return [ siteName ? siteName[ 0 ] : '', topLevelDomain ? topLevelDomain[ 0 ] : '' ];
 }
 
-function getChecklistCompletionProgress( tasks: Task[] | null ) {
+function getTasksProgress( tasks: Task[] | null ) {
 	if ( ! tasks ) {
-		return;
+		// TODO: How do we want to handle that?
+		return [ null, null ];
 	}
 
-	const totalCompletedTasks = tasks.reduce( ( total, currentTask ) => {
+	const completedTasks = tasks.reduce( ( total, currentTask ) => {
 		return currentTask.completed ? total + 1 : total;
 	}, 0 );
 
-	return Math.round( ( totalCompletedTasks / tasks.length ) * 100 );
+	const totalTasks = tasks.length;
```

Comment:
Worth noting that calculating and returning totalTasks here might be a bit redundant. You could already get that directly from from enhancedTasks.length without this method. That said, it's also tidy doing it this way, so works for me.

Reply:
@edanzer That is a great suggestion and simplifies the function a lot (returning a single value instead of an array)